MonoGame.Framework/Graphics/SpriteBatcher.cs

```diff
@@ -108,12 +123,9 @@ public SpriteBatcher (GraphicsDevice device)
         /// <returns></returns>
         public SpriteBatchItem CreateBatchItem()
         {
-            SpriteBatchItem item;
-            if (_freeBatchItemQueue.Count > 0)
-                item = _freeBatchItemQueue.Dequeue();
-            else
-                item = new SpriteBatchItem();
-            _batchItemList.Add(item);
+            if (_batchItemCount >= _batchItemList.Count)
+                _batchItemList.Add(new SpriteBatchItem());
+            var item = _batchItemList[_batchItemCount++];

```

Comment:
Ok, done. 
This of course affects the benchmark results because the first frame takes longer than before.